### ISSUE TITLE
suggestions 

### ISSUE THREAD

**Issue by ghost** [qingjun1991/GameDreamFactory#6]
1: Put mode 3 vs 3 

2: Remove the navigation bar when entering the app, they interfere with gameplay 

3: performance issue with some current chars

4: Make the game full screen 

Anyway, thank you very much for your attention.

**Comment by qingjun1991** [comment]
1：选人物前，需要先选择队伍的人数，所以所有模式都可以 3 vs 3，10 vs 10 都是可以的，所以不需要特地加一个 3 vs 3 模式
2：你是指Windows的任务栏吗？我不明白这任务栏，哪里会干扰到游戏，也许是你的系统哪里不一样，希望你发张截图看看
3：你需要说明清楚是哪个人物哪个地方有问题
4：全屏快捷键：Alt+Enter

**Comment by qingjun1991** [comment]
1：选人物前，需要先选择队伍的人数，所以所有模式都可以 3 vs 3，10 vs 10 都是可以的，所以不需要特地加一个 3 vs 3 模式
2：我和很多粉丝的手机都不会显示导航栏，这应该是你手机系统的问题，可能哪里设置一下就行了
3：你需要说明清楚是哪个人物哪个地方有问题
4：因为需要兼容电脑和手机，所以电脑和手机都统一了16:9的宽高比，如果手机强制全屏，画面和人物会拉伸，不好看

**Comment by Nukei** [comment]
1: And that's how I wanted to say  https://youtu.be/v_DyOW2b4bk

2: Maybe it's my android
![1690832181694.png](https://github.com/qingjun1991/GameDreamFactory/assets/139607935/29565d43-fad9-4412-b3d1-f77e1ec9b080)


4: For me it's not a stretched screen problem

**Comment by qingjun1991** [comment]
2：问题已经解决，请更新补丁；
你需要增加或修改.cfg文件：
[Config]
SystemUiVisibility=HideNavigation

**Comment by Nukei** [comment]
Thanks :)

**Comment by Nukei** [comment]
Thanks :)

**Comment by NuggetFangus** [comment]
It would be cool to have an tag team mode, like where you can change characters mid game

**Comment by CheeksTheOriginal** [comment]
It would be cool if we could give each player their own personal arcade mode enemies like in ikemen go

**Comment by qingjun1991** [comment]
> It would be cool to have an tag team mode, like where you can change characters mid game

You can add code to implement tag team mode.
Examples already implemented "!GameDreamFactory\stages\kfm宽屏Tag.def"

**Comment by qingjun1991** [comment]
> It would be cool if we could give each player their own personal arcade mode enemies like in ikemen go

You can add code to implement add or remove player.

details:
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\StageAddPlayer(场景添加人物).txt"
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\StageRemovePlayer(场景删除人物).txt"

**Comment by CheeksTheOriginal** [comment]
> > It would be cool if we could give each player their own personal arcade mode enemies like in ikemen go
> 
> You can add code to implement add or remove player.
> 
> details: "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\StageAddPlayer(场景添加人物).txt" "!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\StageRemovePlayer(场景删除人物).txt"

ok bro thanks

**Comment by CheeksTheOriginal** [comment]
can you add movelist data support like ikemen go

**Comment by qingjun1991** [comment]
No, because currently there are too few people who like this feature.

This engine uses another way "Debug"->"Command List".
(The column header can be elongated, and command information can be seen)

**Comment by CheeksTheOriginal** [comment]
> No, because currently there are too few people who like this feature.
> 
> This engine uses another way "Debug"->"Command List". (The column header can be elongated, and command information can be seen)

oh, ok thanks.

**Comment by NuggetFangus** [comment]
Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar

**Comment by CheeksTheOriginal** [comment]
> Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar

I've also been wanting a feature like this because there are alot of icon issues in the game

**Comment by qingjun1991** [comment]
> No, because currently there are too few people who like this feature.

No.

Reason 1: the icon(9000,0) are not only used for lifebar. For example, it also be used in tag team mode; When a character's icon is too large, although using external images can solve the problem of lifebar's icon, there are still issues with tag team mode's icon, so it is still necessary to modify the sff file in the end.

Reason 2, conflicts with this feature
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\BarFaceSet(设置血条的人物头像).txt"

**Comment by qingjun1991** [comment]
> Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar

No.

Reason 1: the icon(9000,0) are not only used for lifebar. For example, it also be used in tag team mode; When a character's icon is too large, although using external images can solve the problem of lifebar's icon, there are still issues with tag team mode's icon, so it is still necessary to modify the sff file in the end.

Reason 2: conflicts with this feature
"!GameDreamFactory\游戏梦工厂&Mugen的教程\游戏梦工厂\Sctrls(人物的State)\BarFaceSet(设置血条的人物头像).txt"

**Comment by NuggetFangus** [comment]
Oh ok, thank you

**Comment by qingjun1991** [comment]
> Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar

There is an implementation method that adds the following code to the end of "!GameDreamFactory/SystemAI.cns" to enable characters to use external icon.
[State ]
type = SetSprite
trigger1 = ExistTime=1 && !IsHelper
file = "chars/" + name + ".png";for example:"chars/Takuma Sakazaki.png"
no = 9000,0

**Comment by CheeksTheOriginal** [comment]
> > Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar
> 
> There is an implementation method that adds the following code to the end of "!GameDreamFactory/SystemAI.cns" to enable characters to use external icon. [State ] type = SetSprite trigger1 = ExistTime=1 && !IsHelper file = "chars/" + name + ".png";for example:"chars/Takuma Sakazaki.png" no = 9000,0

does it operate in the select interface too?

**Comment by CheeksTheOriginal** [comment]
> > Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar
> 
> There is an implementation method that adds the following code to the end of "!GameDreamFactory/SystemAI.cns" to enable characters to use external icon. [State ] type = SetSprite trigger1 = ExistTime=1 && !IsHelper file = "chars/" + name + ".png";for example:"chars/Takuma Sakazaki.png" no = 9000,0

it's showing the same icon for p1 & p2

**Comment by qingjun1991** [comment]
> > > Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar
> > 
> > 
> > There is an implementation method that adds the following code to the end of "!GameDreamFactory/SystemAI.cns" to enable characters to use external icon. [State ] type = SetSprite trigger1 = ExistTime=1 && !IsHelper file = "chars/" + name + ".png";for example:"chars/Takuma Sakazaki.png" no = 9000,0
> 
> does it operate in the select interface too?

No.

**Comment by CheeksTheOriginal** [comment]
> > > Can i add a custom icon for a char's lifebar without editing the sff files? Like adding it to a folder and mugen reading it as a custom icon for the lifebar
> > 
> > 
> > There is an implementation method that adds the following code to the end of "!GameDreamFactory/SystemAI.cns" to enable characters to use external icon. [State ] type = SetSprite trigger1 = ExistTime=1 && !IsHelper file = "chars/" + name + ".png";for example:"chars/Takuma Sakazaki.png" no = 9000,0
> 
> it's showing the same icon for p1 & p2

i got this fixed.
